Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,24 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
+  const result = tasks
+    .map(item => item.duration / 60)
+    .filter(item => item >= 2)
+    .reduce((totalRate, item) => totalRate + item * hourlyRate, 0);
   console.log(tasks, hourlyRate);
+  console.log(result);
+  return result;
 }
 
 // eslint-disable-next-line no-unused-vars
 const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
 
 // add code to convert `earnings` to a string rounded to two decimals (euro cents)
-
-console.log(`Maartje has earned €${'replace this string with the earnings rounded to euro cents'}`);
+const roundedEarnings = earnings
+  .toFixed(2)
+  .toString()
```

Comment:
One more thing: `.toFixed()` already returns a string so you don't need the `.toString()` here. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed